docs/docs/Contributing/contributing-how-to-contribute.md

```diff
@@ -18,6 +21,11 @@ Install Langflow from source by forking the repository, and then set up your dev
 * [Node.js](https://nodejs.org/en/download/package-manager)
 * [Make](https://www.gnu.org/software/make/#documentation)
 
+:::important
+Make is not required for Windows installations. Installing and running Windows from source uses node and uv.
+Scripts are also provided in the `/scripts/windows` directory of the Langflow project.
+:::
+
```

Comment:
```suggestion
* [Make](https://www.gnu.org/software/make/#documentation)

:::tip Windows
For Windows installations, you don't need need Make, and you can find [Windows scripts](https://github.com/langflow-ai/langflow/tree/main/scripts/windows) in the Langflow repository.
:::
```

Important is a little strong here.
Another alternative would be to use tabs (One tab for Linux and macOS, one tab for Windows) with the prereqs/info for each OS on each tab.